frontend/Dockerfile.dev

```diff
@@ -0,0 +1,4 @@
+FROM node:16-alpine
```

Comment:
when i made the file, i thought the same thing, its a bit yucky. however its in this particular spot to make docker/compose build "just work" and/or work with ease. if there is a way to get the build to work nice with disjointed assets, i am also on board, but it might be rather confusing without the docker-compose.yaml as a reference.